examples/transaction-0.json

```diff
@@ -223,7 +223,7 @@
           {
             "url": "https://fhir.bbmri.de/StructureDefinition/Custodian",
             "valueReference": {
-              "reference": "Organization/collection-2"
+              "reference": "http://blaze.example.com/Organization/collection-2"
```

Comment:
here I would append the context path `/fhir` because Blaze works with it.

Reply:
Done. I also added some Ducumentation that uses /fhir in its examples.